plugin.py

```diff
@@ -35,16 +33,17 @@ def zipfile(cls) -> str:
         return os.path.join(cls.basedir(), "lua.vsix")
 
     @classmethod
-    def binplatform(cls) -> str:
+    def platform_arch(cls) -> str:
         return {
-            "linux": "Linux",
-            "windows": "Windows",
-            "osx": "macOS"
-        }[sublime.platform()]
+            "linux_x64": "linux-x64",
+            "osx_arm64": "darwin-arm64",
+            "osx_x64": "darwin-x64",
+            "windows_x64": "win32-x64",
+        }[sublime.executable_hash()[1]]
```

Comment:
Maybe instead of using undocumented API do "sublime.platform() - sublime.arch()" instead?